src/lib/draw/AttributeMode.svelte

```diff
@@ -17,12 +16,12 @@
 
   export function start() {}
   export function stop() {
-    setCurrentlyEditing(null);
+    formOpen.set(null);
   }
 
   // Calculate hover
   $map.on("mousemove", (e) => {
-    if (mode == thisMode && $currentlyEditing == null) {
```

Comment:
This used to be necessary for behavior (don't hover another object when a form is open). Now we take care of that in `HoverLayer` better. We could still skip this calculation as an optimization, but it doesn't seem to matter much